Assets/MixedRealityToolkit/Utilities/MixedRealityPlayspace.cs

```diff
@@ -39,34 +39,43 @@ public static Transform Transform
         {
             get
             {
-                if (mixedRealityPlayspace)
+                if (mixedRealityPlayspace != null)
                 {
                     mixedRealityPlayspace.gameObject.SetActive(true);
                     return mixedRealityPlayspace;
                 }
 
-                if (CameraCache.Main.transform.parent == null)
-                {
-                    // Create a new mixed reality playspace
+                if (MixedRealityToolkit.IsCameraSystemEnabled)
+                {   // If the camera system is enabled, let it handle parenting the camera.
+                    // Just create a transform.
                     GameObject mixedRealityPlayspaceGo = new GameObject(Name);
                     mixedRealityPlayspace = mixedRealityPlayspaceGo.transform;
-                    CameraCache.Main.transform.SetParent(mixedRealityPlayspace);
                 }
                 else
                 {
-                    if (CameraCache.Main.transform.parent.name != Name)
+                    if (CameraCache.Main.transform.parent == null)
```

Comment:
We should be careful about blanket == null operators like this as Unity "may" remove them
https://blogs.unity3d.com/2014/05/16/custom-operator-should-we-keep-it/
It's unclear as to what is the best alternative as yet.

Reply:
I've been taking the (possibly wreckless) approach of ignoring this possibility until they actually follow through. Because the consequences would be so far-reaching that I see no point in trying to get ahead of cleaning up the rubble.

Reply:
@SimonDarksideJ, that Unity blog post was from five years ago. I concur with @Railboy that we should not over index on something that was asked and no related product change for that time span.